.golic.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Absa Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+golic:
+  licenses:
+    apache2: |
+        Copyright {{copyright}}
+
+        Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+
+        Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+    copyright: |
+        Copyright © {{copyright}}
+
+        All rights reserved
+
+        Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+  rules:
+    .go:
+      prefix: "/*"
+      suffix: "*/"
+    .gotxt:
+      prefix: "/*"
+      suffix: "*/"
+    .java:
+      prefix: "/*"
+      suffix: "*/"
+    .scala:
+      prefix: "/*"
+      suffix: "*/"
+    .cs:
+      prefix: "/*"
+      suffix: "*/"
+    .html:
+      prefix: "<!--"
+      suffix: "-->"
+    .js:
+      prefix: "/*"
+      suffix: "*/"
+    .css:
+      prefix: "/*"
+      suffix: "*/"
+    .yaml:
+      prefix: "#"
+    .yml:
+      prefix: "#"
+    Dockerfile*:
+      prefix: "#"
+    Makefile:
+      prefix: "#"
+    .gitignore:
+      prefix: "#"
+    .licignore:
+      prefix: "#"
+    .tf:
+      prefix: "#"
+    .sh:
+      prefix: "#"
+      under:
+        - "!/bin/sh"
+        - "!/bin/bash"
+        - "!/bin/ksh"
+        - "!/bin/awk"
+        - "!/bin/expect"
+        - "!/bin/csh"
+        - "!/usr/bin/perl"
+        - "!/usr/bin/python"
+        - "!/usr/bin/ruby"
+        - "!/usr/bin/php"
+    .xml:
+      prefix: "<!--"
+      suffix: "-->"
+      under:
+        - "<?xml"
```

Comment:
What about keeping only diff + license locally, the rest will stay online in masterconfig ? 
```yaml
golic:
  licenses:
    apache2: |
        Copyright {{copyright}}

        Licensed under the Apache License, Version 2.0 (the "License");
        you may not use this file except in compliance with the License.
        You may obtain a copy of the License at

            http://www.apache.org/licenses/LICENSE-2.0

        Unless required by applicable law or agreed to in writing, software
        distributed under the License is distributed on an "AS IS" BASIS,
        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
        See the License for the specific language governing permissions and
        limitations under the License.

        Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
  rules:
    .txt:
      prefix: ""
      indent: "NO_INDENT" # I added this after yesterday call (we can remove all prefixes and empty spaces), see:  https://github.com/AbsaOSS/golic/releases/tag/v0.4.2
``

Reply:
Excellent @kuritka ! If it works that way - only happy to amend!

Reply:
@kuritka amended pls check